### ISSUE TITLE
`@edit` not working on remote

### ISSUE THREAD

**Issue by marius311** [julia-vscode/julia-vscode#2216]
On local VSCode sessions called `@edit` from VSCode either inline or from the REPL correctly opens the appropriate file in VSCode itself, but in a Remote SSH session I get eg

```
julia> @edit 1+2

julia> /global/u1/m/marius/src/julia-1.6.1/share/julia/base/int.jl:1
# This file is a part of Julia. License is MIT: https://julialang.org/license
^

SyntaxError: Invalid or unexpected token
    at wrapSafe (internal/modules/cjs/loader.js:979:16)
    at Module._compile (internal/modules/cjs/loader.js:1027:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
```

which likely stems from the fact that 
```
julia> ENV["JULIA_EDITOR"]
"\"/global/u1/m/marius/.vscode-server/bin/b4c1bd0a9b03c749ea011b06c6d2676c8091a70c/node\""
```

I could be wrong but I feel like this used to work in older versions (maybe pre 1.2?). 

I'm on julia-vscode 1.2.4-insider and VSCode 1.57.0.

**Comment by johannes-fischer** [comment]
I can confirm this issue, just had the same experience.

julia-vscode v1.5.8, vscode 1.63.1

**Comment by albheim** [comment]
I was a little curious and tried to understand what was going on here.

First I found 
https://github.com/julia-vscode/julia-vscode/blob/114188bbab8ad8b36318aabe00e1d78d4ed5f783/src/interactive/repl.ts#L92-L106
which seems to be the source of setting the `JULIA_EDITOR` when starting the internal REPL.

So I assume I don't want to have the `julia.editor` setting set to anything since then I would override this?

Opening a SSH remote and starting a terminal in it, I can see that the `PATH` has some `$HOME/.vscode-server/bin/<some hash>/bin` at the front, which seems connected to this specific session (or maybe it is for all session connected with this specific version if <hash> is some specific commit?).

In bin there we had 
```
bin
├── code-server
├── helpers
│   └── browser.sh
└── remote-cli
    └── code
```
where `code-server` seemed to start some service listening to some port, not really what I wanted. But the `remote-cli/code` seemed to do the desired thing, running `.vscode-server/bin/<some-hash>/bin/remote-cli/code /some/path/to/file` it opens the file. But `code` here seems to not be in the path since only the `bin` directory was added and not the subdirectories.

Moving `code` to above folder and fixing the `ROOT` path in the script still gives me the same error though.

Checking the actual value set (should have done this first...) I find that
```julia
julia> ENV["JULIA_EDITOR"]
"\"/local/home/albheim/.vscode-server/bin/d6ee99e4c045a6716e5c653d7da8e9ae6f5a8b03/node\""
```
which seems to be the problem. Here I assume I must have gotten this from line 102 above, 
```
return `"${process.execPath}"`
```
where this is the running process and does not seem to be the same process that can be used to open files?

I guess I could add `"julia.editor": ".vscode-server/bin/<some-hash>/bin/remote-cli/code"` as a setting for only this remote worker or something, but this would then need to be updated every time I update vscode?

I did read some other answers about vscode remotes, so I think this might be something that is changing currently. Maybe not possible to get a solution that works well over different versions?

For my version
```
Version: 1.64.1
Commit: d6ee99e4c045a6716e5c653d7da8e9ae6f5a8b03
Date: 2022-02-07T17:17:47.733Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Linux x64 5.13.13-200.fc34.x86_64
```
it seems like replacing line 102 with something that did something like `return "$(dirname ${process.execPath})/bin/remote-cli/code"` (but with correct code...)

TLDR
* Seems like vscode has a `code` command that works, but it is not added to path (vscode problem?).
* Seems like julia extension sets `.../node` as the julia editor which does not seem to work, for my version it should probably be `.../bin/remote-cli/code` (or just `code` if it was correctly added to path) but not sure if this is true in general.
